packages/sit-onyx/src/components/OnyxSwitch/OnyxSwitch.ct.tsx

```diff
@@ -1,48 +1,166 @@
+import { createScreenshotsForAllStates } from "../../utils/playwright";
 import { expect, test } from "../../playwright-axe";
 import OnyxSwitch from "./OnyxSwitch.vue";
 
-test("should render default OnyxSwitch", async ({ mount, makeAxeBuilder }) => {
+test("should render unchecked", async ({ mount, makeAxeBuilder }) => {
   // ARRANGE
-  const component = await mount(<OnyxSwitch label="Switch" />);
-
-  // ASSERT
-  await expect(component).toContainText("Switch");
-  await expect(component).toHaveScreenshot("default.png");
+  const component = await mount(<OnyxSwitch label="Unchecked" />);
 
   // ACT
   const accessibilityScanResults = await makeAxeBuilder().analyze();
 
   // ASSERT
   expect(accessibilityScanResults.violations).toEqual([]);
-});
 
-test("should render disabled OnyxSwitch", async ({ mount, makeAxeBuilder }) => {
-  // ARRANGE
-  const component = await mount(<OnyxSwitch label="Switch" disabled />);
+  const onyxSwitch = component.getByRole("checkbox");
+
+  // ACT
+  await onyxSwitch.hover();
 
   // ASSERT
-  await expect(component).toContainText("Switch");
-  await expect(component).toBeDisabled();
-  await expect(component).toHaveScreenshot("disabled.png");
+  await expect(onyxSwitch).not.toBeChecked();
```

Comment:
```suggestion
  await expect(component).not.toBeChecked();
```

You can also directly check the component because it is the `<label>` element which is equivalent :) Same for the other tests